client/blocks/comment-detail/comment-author-meta.jsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+
+export const CommentAuthorMeta = ( {
+	author,
+	translate,
+} ) =>
+	<div className="comment-detail__author-meta">
+		<div>
+			<Gridicon icon="link" size={ 18 } />
+			{ author.URL }
+		</div>
+		<div>
+			<Gridicon icon="mail" size={ 18 } />
+			{ author.email }
+		</div>
+		<div>
+			<Gridicon icon="globe" size={ 18 } />
+			{ author.ip }
+		</div>
+		<div>
+			<Gridicon icon="comment" size={ 18 } />
+			{ translate(
+				'%d Comment', '%d Comments',
```

Comment:
Hi! I've found a possible matching string that has already been translated **37** times:
`translate( '%d comment', '%d comments' )` *ES Score: 20*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/comments/detail-view)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).